packages/flame/lib/src/game/flame_game.dart

```diff
@@ -182,4 +184,9 @@ class FlameGame extends ComponentTreeRoot with Game {
       }
     }
   }
+
+  @override
+  set size(Vector2 value) {
+    throw UnsupportedError('The size of FlameGame cannot be set explicitly');
+  }
```

Comment:
would it be beneficial to create another interface fot things that can only get the size?
like:
```dart
abstract class ReadableSizeProvider {
  Vector2 get size;
}
abstract class SizeProvider extends ReadableSizeProvider {
  // ...
}
```
then each class can implement specifically what they have, and things are a bit more type safe

Reply:
Good idea